compiler/q-implant/README.md

```diff
@@ -2,13 +2,57 @@
 
 _q-implant_ writes quantization parameters and weights (given as .json and .npy files) to a circle model.
 
+## Format of input files (`.json` and `.npy`)
+
+The main input file is a `.json` file, which is a dictionary.
+
+The dictionary's key is a tensor name, and its value is quantization parameters and values(if exists).
```

Comment:
limit in 80/100 cols is for better editing. long horizontal text needs horizontal scrolling which is a pain in editing. I didn't say to change the text. just split the text.

this suggestion is something what I wanted.
```suggestion
The main input file is a `.json` file, which is a dictionary whose key is a tensor name
and value is quantization parameters and values (if exists).
```

Reply:
and plz split like this for all other lines.